src/components/NewClusterModal.js

```diff
@@ -212,12 +213,13 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
 
     const makeGroupedEnvSelect = id => h(GroupedSelect, {
       id,
+      'aria-label': 'Select Environment',
```

Comment:
This select looks like it already has a properly wired up label, so we should be able to use that instead of adding an `aria-label`